bats/no-repo.bats

```diff
@@ -226,6 +226,6 @@ NOT_VALID_REPO_ERROR="The current directory is not a valid dolt repository."
 @test "all versions of help work outside a repository" {
     dolt checkout --help
     dolt checkout -help
-    skip "No dashes in front of help segfaults right now"
-    dolt checkout help
+    run dolt checkout help
+    [ "$status" -ne 0 ]
```

Comment:
You can just remove the run and the `[ "$status" -ne 0 ]`. Running a command naked just checks for $status -eq 0

Reply:
It won't be equal to 0.  It'll exit non-zero which is what I'm checking